apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -474,6 +508,60 @@ void _wt_connect(void *arg)
 	WT_LEAVE;
 }
 
+void _wt_scan_connect(void *arg)
+{
+	WT_ENTER;
+	WT_LOG(TAG, "--> scan_connect\n");
+	g_scan_join = 1;
+	struct wt_options *ap_info = (struct wt_options *)arg;
+	wifi_manager_scan_config_s sconfig;
+	sconfig.channel = 0;
+	sconfig.ssid_length = strlen(ap_info->ssid);
+	strncpy(sconfig.ssid, ap_info->ssid, sconfig.ssid_length + 1);
+	strncpy(g_scanned_ssid, ap_info->ssid, sconfig.ssid_length + 1);
+
+	wifi_manager_result_e res = wifi_manager_scan_ap(&sconfig);
+
+	if (res != WIFI_MANAGER_SUCCESS) {
+		WT_LOGE(TAG, "request scan ssid (%s) fail %d\n", sconfig.ssid, res);
+		return;
+	}
+	WT_TEST_WAIT;
+
+	g_scan_join = 0;
+	if (g_scanned_result != 0) {
```

Comment:
`g_scanned_result` is initialized as `0` in line 190, `static int g_scanned_result = 0;`
But the result value for checking success is `0`.
So this can be passed although _wt_sta_connected is not executed.
I think `g_scanned_result` should be initialized as `-1`.

Reply:
g_scanned_result is only set by calling _wt_get_scanned_list(). So I think it's ok.